### PR TITLE
NFC: Add Security section to README

### PR DESCRIPTION
**Pull request by bennbollay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +161,20 @@ Don't see the service you are looking for? We are constantly adding support for
...
+as-a-Service ("iPaaS") that provides a broad and full-featured integration platform.  All data stored in
+Fusebit is:
+  * Encrypted at rest using AES-256 encryption
+  * Encrypted in transit using TLS 1.2
+  * Secured by a powerful authorization engine that powers the Fusebit integration platform itself, trusted
+  and used already in production by Fusebit customers
+  * Identity backed by either private keys or OAuth 2
+
+Fusebit has a strong committment to maintain a high standard for security practices, and welcomes any bug
+reports or security advisories.  Please contact support@fusebit.io for any further questions.
```
Comment (degrammer) [inline README.md:176]: Fusebit is committed to maintaining a high standard of security practices and welcomes any bug reports or security advisories. Don't hesitate to get in touch with support@fusebit.io for any further questions.

**Review by degrammer — CHANGES_REQUESTED** [review]
Check grammar suggestion.

File: README.md (hunk 1 above)
Comment (bennbollay) [reply README.md]: Done.

**Review by degrammer — APPROVED** [review]
LGTM

**Pull request merged**